web/src/beta/lib/reearth-ui/components/Button/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { FC, useCallback, useState } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+export type ButtonProps = {
+  appearance?: "primary" | "secondary" | "dangerous" | "simple";
+  disabled?: boolean;
+  size?: "normal" | "small";
+  iconButton?: boolean;
+  icon?: string; // TODO: Icon Name, Use Icon Component
+  title?: string;
+  onClick?: () => void;
+};
+
+export const Button: FC<ButtonProps> = ({
+  appearance = "secondary",
+  icon = "",
+  disabled = false,
+  size = "normal",
+  iconButton = false,
+  title = "",
```

Comment:
I think we should only give default value to props when necessary, in this case i feel `icon` `disabled` `iconButton` `title` don't require a default value. What do you think?